src/pages/index.js

```diff
@@ -67,36 +97,152 @@ addNewCardButton.addEventListener("click", () => {
   newCardPopup.open();
 });
 
-const profilePopup = new PopupWithForm("#edit-modal", (formData) => {
+//=========================
+//=== Get Profile Popup ===
+//=========================
+api
+  .getUserInfo()
+  .then((userData) => {
+    userInfo.setUserInfo(userData);
+    userInfo.setUserAvatar(userData);
+  })
+  .catch((error) => {
+    console.error("info not fetched from the server", error);
+  });
+
+//===========================
+//=== Edit Profile Popup ====
+//===========================
+const profilePopup = new PopupWithForm("#edit-profile-modal", (formData) => {
   const name = formData["name"];
   const job = formData["description"];
+  profilePopup.setButtonText(true);
+  api
+    .updateProfile(name, job)
+    .then((updatedData) => {
+      userInfo.setUserInfo({
+        name: updatedData.name,
+        about: updatedData.about,
+      });
+      addCardValidator.disableSubmitButton();
```

Comment:
`addCardValidator.disableSubmitButton()` is for the `card` form. You need to use the `profile validator` here